### ISSUE TITLE
[1.10.2-2.0.0.105] Did you forget to remove the easier end crystal recipe?

### ISSUE THREAD

**Issue by RealGrep** [brandon3055/Draconic-Evolution#404]
## Description
I have the default harder recipe for the end crystal, but I also see the old one still in there. Restarted the server and client to be sure, and yep, still there.

So as the subject says, did you forget to remove the old recipe when it's set to expensiveDragonRitual=true? XD

Super easy to fix with CraftTweaker, but thought you'd like to know it still appears.

## Base information
* Minecraft version: 1.10.2
* Minecraft Forge version: 2095
* Draconic Evolution version: 2.0.0.105

--- Stuff that is almost certainly irrelevant in this case snipped, but if you need more, let me know ---

**Comment by brandon3055** [comment]
Ok thats odd... I do remove all recipes that produce an EnderCrystal before adding my own and that worked in dev so i have no idea whats going on there... Do you use any other mods that modify the recipe list?

**Comment by RealGrep** [comment]
Hm, well, there's CraftTweaker, but there's no recipes for that in my scripts. Looks like the mod list is relevant after all: https://www.dropbox.com/s/b8nc869qwrjzkh9/ModList%20%28Case%20Conflict%29.txt?dl=0

I went through the list of them. None really deal with the dragon, afaik. Closest is "Bad Wither No Cookie" that stops the sound from being broadcast when he dies. Weird. Figured the simplest explanation was just forgetting to remove the recipe first, but hello, another weird bug.

Well, at least you disabled the random bug generator. ;)

Ok, going through the FML log on the server, I do see this:
[Server thread/DEBUG] [EnderIO/EnderIO]: Registering XML recipe 'Compatibility: End Crystal'
[Server thread/DEBUG] [EnderIO/EnderIO]: Registering ShapedOreRecipe: 1xitem.end_crystal@0: [ABC, DEF, GHI, A, blockGlassColorless, B, blockGlassColorless, C, blockGlassColorless, D, blockGlassColorless, E, pearlEnderEye, F, blockGlassColorless, G, blockGlassColorless, H, itemGhastTear, I, blockGlassColorless]

That looks like a likely candidate for the problem.

Here's that full log for your perusal: https://www.dropbox.com/s/twnho507ozsv55x/fml-server-latest_FORDE2.log?dl=0

I can do some tests somewhat later, but yeah, looks like EIO is adding that recipe, for some reason.
